### PR TITLE
Support SRI

### LINKED ISSUE #72: Support SRI - Subresource Integrity
The URLs to Bootstrap, jQuery and Popper should contain (ideally by default) the SRI hashes.
From [MDN](https://developer.mozilla.org/en-US/docs/Web/Security/Subresource_Integrity#Tools_for_generating_SRI_hashes):

> Subresource Integrity (SRI) is a security feature that enables browsers to verify that resources they fetch (for example, from a CDN) are delivered without unexpected manipulation. It works by allowing you to provide a cryptographic hash that a fetched resource must match.

On [Bootstrap's download page](https://getbootstrap.com/docs/4.5/getting-started/download/#bootstrapcdn) the SRI tags are included by default for all 3 libraries:
Capture d’écran de 2020-06-08 14-32-33

Example of how another Flask plugin addressed this :https://github.com/miguelgrinberg/Flask-Moment#subresource-integritysri
(in implementing this, it would probably be wise to include an option to deactivate SRI links)

Handy tool to calculate SRI hashes (if they are not listed anymore, for example if still pointing to an old version): https://www.srihash.org/

### PR DESCRIPTION
**Pull request by yuxiaoy1**
Fix #72.

### CODE REVIEW

**Comment by yuxiaoy1** [comment]
The latest commit passed the flake8 test locally on my PC, why it's failed in the CI...
```
(venv) bootstrap-flask $tox -e flake8                                                                                sri
flake8 installed: flake8==3.9.2,importlib-metadata==4.5.0,mccabe==0.6.1,pycodestyle==2.7.0,pyflakes==2.3.1,typing-extensions==3.10.0.0,zipp==3.4.1
flake8 run-test-pre: PYTHONHASHSEED='3748920659'
flake8 run-test: commands[0] | flake8 flask_bootstrap tests
_______________________________________________________ summary ________________________________________________________
  flake8: commands succeeded
  congratulations :)
```

File: flask_bootstrap/__init__.py [hunk 1]
```diff
@@ -84,26 +88,38 @@ def load_css(version=VERSION_BOOTSTRAP):
...
+                           (version, css_filename))
             else:
-                css = '<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootswatch@%s/dist/%s/%s"' \
-                    ' type="text/css">' % (version, bootswatch_theme.lower(), css_filename)
+                css = ('<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootswatch@%s/dist/%s/%s">' %
+                       (version, bootswatch_theme.lower(), css_filename))
         return Markup(css)
 
-    @staticmethod
+    @staticmethod  # noqa: C901
```
Comment (Demetriex) [inline flask_bootstrap/__init__.py:119]:
```
flask_bootstrap/__init__.py:120:5: C901 'Bootstrap.load_js' is too complex (15)
```

The inline ignore should be placed on line 120

Comment (yuxiaoy1) [reply flask_bootstrap/__init__.py:119]: I saw that, it shows line 119 on my PC before, so I add the ignoration to line 119 and then it passed the test.
The inconsistency between my PC and the CI runner is kind of odd, will try to find it out later.

Comment (yuxiaoy1) [reply flask_bootstrap/__init__.py:119]: I moved the ignoration from line 119 to 120, then it failed the test on my PC...
@greyli any suggestions?
```
(venv) bootstrap-flask $tox -e flake8                                                                                                            sri*
flake8 installed: flake8==3.9.2,importlib-metadata==4.5.0,mccabe==0.6.1,pycodestyle==2.7.0,pyflakes==2.3.1,typing-extensions==3.10.0.0,zipp==3.4.1
flake8 run-test-pre: PYTHONHASHSEED='304898279'
flake8 run-test: commands[0] | flake8 flask_bootstrap tests
flask_bootstrap/__init__.py:119:5: C901 'Bootstrap.load_js' is too complex (15)
ERROR: InvocationError for command /home/yuxiaoy/python/oss/bootstrap-flask/.tox/flake8/bin/flake8 flask_bootstrap tests (exited with code 1)
______________________________________________________________________ summary ______________________________________________________________________
ERROR:   flake8: commands failed
```

Comment (greyli) [reply flask_bootstrap/__init__.py]: Not sure what happened here, I just move the ignore comment to L120...

**Comment by greyli** [comment]
Merged, thanks!

**Pull request merged**